ksp_plugin_adapter/orbit_analyser.cs

```diff
@@ -236,8 +244,24 @@ internal class OrbitAnalyser : VesselSupervisedWindowRenderer {
     UnityEngine.GUI.DragWindow();
   }
 
-  private void RenderOrbitalElements(OrbitalElements? elements) {
-      UnityEngine.GUILayout.Label("Orbital elements");
+  private void RenderOrbitalElements(OrbitalElements? elements,
+                                     CelestialBody primary) {
+      double? lowest_distance = elements?.distance.min;
+      LabeledField(
+          "Lowest altitude",
+          (lowest_distance - primary?.Radius)?.FormatAltitude());
+      double? lowest_primary_altitude =
+          primary?.ocean == true ? 0 : primary?.pqsController?.radiusMin;
+      string altitude_warning = lowest_distance < lowest_primary_altitude
```

Comment:
What happens in this comparison if one (or both) of the distances doesn't have a value?

Reply:
`false`.